src/index.html

```diff
@@ -1,13 +1,252 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
We have only one html tag on the website, you can style him only.

Reply:
it's not at all clear what's wrong

Reply:
This is not good practice, for example for html 4 this syntax is not valid. For HTML 5 is a valid case, but theoretically, if somebody uses old browser they will see the error.
In principle, this will not be the biggest problem for the use / old browser, but nevertheless.